src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeLibrary.cs

```diff
@@ -100,8 +101,8 @@ public static IntPtr Load(string libraryName, Assembly assembly, DllImportSearch
         /// NativeLibrary Loader: High-level API that doesn't throw.
         /// </summary>
         /// <param name="libraryName">The name of the native library to be loaded</param>
```

Comment:
The notes within the summary section for `NativeLibrary.Load` should be repeated here, so that documentation page picks it up,